frontend/src/container/Header.jsx

```diff
@@ -40,7 +40,6 @@ const qs = require('query-string')
 
 export class Header extends React.Component {
   componentDidMount () {
-    this.props.dispatchCustomEvent('TRACIM_HEADER_MOUNTED', {})
```

Comment:
why are you removing this ? I haven't seen you adding it back somewhere else

Reply:
It's not used anywhere as far as I could see. It's therefore unnecessary and untested code, and I was trying to change unneeded state changes in this PR.

Should I revert this anyway?

Reply:
It is useful, keep it.
The purpose of this is to allow external apps to do stuffs based on Tracim's event

Reply:
all right! I'm going to revert this.

I have a question however: isn't this prematurilly guessing what will be needed in an uncertain future on a architecture that may change sooner or later anyway?